simd/unit_tests/include/SIMDTesting_Ops.hpp

```diff
@@ -524,15 +534,18 @@ class masked_reduce {
  public:
   template <typename T, typename U, typename MaskType>
   auto on_host(T const& a, U const& identity, MaskType mask) const {
-    return Kokkos::Experimental::reduce(a, mask, identity, BinaryOperation());
+    return Kokkos::Experimental::reduce(a, mask, BinaryOperation(), identity);
   }
   template <typename T, typename U, typename MaskType>
   auto on_host_serial(T const& a, U const& identity, MaskType mask) const {
     if (Kokkos::Experimental::none_of(mask)) return identity;
     auto w        = Kokkos::Experimental::where(mask, a);
     auto const& v = w.impl_get_value();
     auto const& m = w.impl_get_mask();
-    U result      = Kokkos::Experimental::Impl::Identity<U, BinaryOperation>();
+    U result =
+        (Kokkos::Experimental::Impl::is_basic_reduction_op_v<BinaryOperation>)
```

Comment:
I think that the parentheses here are unnecessary
```suggestion
        Kokkos::Experimental::Impl::is_basic_reduction_op_v<BinaryOperation>
```

Reply:
Applied.